README.md

```diff
@@ -82,6 +83,9 @@ ebcli tx send validator $(ebcli keys show testuser -a) 10stake --chain-id=peggy
 ebcli query account $(ebcli keys show validator -a) --trust-node
 ebcli query account $(ebcli keys show testuser -a) --trust-node
 
+# Then wait 10 seconds then confirm your validator was created correctly, and has become Bonded status
+ebcli query staking validators --trust-node
```

Comment:
What's this doing here? This PR has a lot of weird commits in it too, seems like it may have gotten mixed up with other functionality?

Reply:
I've changed the base branch to `base/bridge-contracts`, where I've merged the updated bridge contracts on top of `musnit/add-initial-burn-functionality`. It makes the commit history much cleaner and now it should be easier to review these changes.